src/gi-stubs/repository/Gio.pyi

```diff
@@ -4035,45 +4038,64 @@ class InputVector(GObject.GPointer):
     size: int = ...
 
 # override
-class ListModel(GObject.GInterface):
-    def __contains__(self, item: object) -> bool: ...
-    def __getitem__(self, position: int) -> GObject.Object: ...
-    def __iter__(self) -> Iterator[GObject.Object]: ...
+class ListModel(GObject.GInterface, Generic[ItemType]):
+    # override
```

Comment:
i dont think thats necessary, if the class itself has a "override" annotation, then it will be completely ignored by the generator. So no need to add override to each method.